src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import logo from './assets/spacex.png';
+
+const Header = () => (
+  <header className="d-flex">
+    <div className="d-flex logo-title">
+      <img className="logo" src={logo} alt="spacex logo" />
+      <h1><b>Space Travelers&apos; Hub</b></h1>
+    </div>
+    <nav className="d-flex nav-links">
+      <NavLink to="/">Rockets</NavLink>
+      <NavLink to="/Missions">Missions</NavLink>
+      <div className="line" />
+      <NavLink to="/Profile">My Profile</NavLink>
```

Comment:
- [ ] As per requirement, the active page should be indicated with an underline and all the page links.

Reply:
![fixed eror](https://user-images.githubusercontent.com/28694196/174396150-7e37c722-1ba4-4a7d-9462-1c89fb665ef2.png)

I have fixed it now. thanks for your kind feedback